src/scripts/main.js

```diff
@@ -1 +1,35 @@
 'use strict';
+
+const links = document.querySelectorAll('.list-item__link');
+// const images = document.querySelectorAll('.gallery__thumb');
+const largeImg = document.querySelector('#largeImg');
+
+for (let i = 0; i < links.length; i++) {
+  links[i].removeAttribute('href');
+
+  let number = '';
+
+  switch (i) {
```

Comment:
This is redundant code, if you will read link.href value you dont need to create sourceLink like this.